src/server/api/mutations/updateServiceManager.js

```diff
@@ -1,4 +1,3 @@
-import { GraphQLError } from "graphql/error";
```

Comment:
I see that GraphQLError was deleted.  Is this functionality found inside @apollo/client elsewhere in the code?  What is the replacement plan for this functionality?

Reply:
Never mind, I reviewed the file and can now see that this import simply wasn't used in any functions in the updateServiceManager.  Is that why you deleted it?